addons/account/tests/common.py

```diff
@@ -634,6 +634,15 @@ def _create_invoice(self, **invoice_args):
             **invoice_args,
         })
 
+    def replace_ignore(self, to_compare):
+        """ Because we put jsons in separate files, we can not use ANY from unittest Mock there, so we can just apply
```

Comment:
@clbr-odoo If you could also have a look.  If you do not think it should be done differently in generic than elsewhere, I could just put it in the l10n_es_edi_verifactu_pos module.